app/lib/common/MarketClasses.js

```diff
@@ -762,8 +859,9 @@ class CallOrder {
         if (newOrder.borrowers.indexOf(order.borrower) === -1) {
             newOrder.borrowers.push(order.borrower);
         }
-        newOrder.to_receive += order.to_receive;
-        newOrder.for_sale += order.for_sale;
+        newOrder.debt += order.debt;
+
+        newOrder.collateral += order.collateral;
```

Comment:
I don't know whether need to use 'max_collateral_for_sale' here.

Reply:
It gets pretty messy when summing orders now, I'll have to think about that one a bit (no pun intended).